### PR TITLE
fix(cli): do not inject script tags due to pages not allowing script tags

### PR DESCRIPTION
**Pull request by michael-siek**
Closes: https://github.com/dequelabs/axe-core-npm/issues/711

### CODE REVIEW

**Review by Zidious — APPROVED** [review]
LGTM - 

Tests seems to be failing though 


Title should be scoped to CLI

**Comment by stephenmathieson** [comment]
> Title should be scoped to CLI

The title should also describe the purpose of the commit, not how the commit does it. Refer to the [SDLC](https://github.com/dequelabs/dev-process/tree/main/git#subject)

File: packages/cli/package.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
     ],
     "statements": 95,
     "branches": 92,
-    "functions": 100,
+    "functions": 94,
```
Comment (Zidious) [inline packages/cli/package.json:81]: nit - This is still above the required coverage, but maintain or increase coverage IMO

**Review by Zidious — APPROVED** [review]
LGTM -  minor nit inline feel free to ignore

File: packages/cli/package.json (hunk 1 above)
Comment (michael-siek) [reply packages/cli/package.json:81]: I also removed test but those test only tested the thing that I removed

**Pull request merged**